dbms/src/Flash/Mpp/MPPTunnel.h

```diff
@@ -279,9 +282,6 @@ class LocalTunnelSender : public TunnelSender
         data->switchMemTracker(local_request_handler.recv_mem_tracker);
 
         auto res = local_request_handler.write<enable_fine_grained_shuffle>(source_index, data);
-
-        if (unlikely(!res))
```

Comment:
why remove here?

Reply:
> why remove here?

I think `push` should not be involved in the status management. Because returning false will trigger the throwing of exception, then MppTask will cancel the tunnel which then call the connectionDone function.

Reply:
ok